tests/virt_autotest/virt_utils.pm

```diff
@@ -16,16 +16,18 @@ use base Exporter;
 use Exporter;
 use strict;
 use warnings;
+use Sys::Hostname;
 use File::Basename;
 use testapi;
 use Data::Dumper;
 use XML::Writer;
 use IO::File;
 use proxymode;
 use version_utils 'is_sle';
+use xen;
 
 our @EXPORT
-  = qw(update_guest_configurations_with_daily_build repl_addon_with_daily_build_module_in_files repl_module_in_sourcefile handle_sp_in_settings handle_sp_in_settings_with_fcs handle_sp_in_settings_with_sp0 clean_up_red_disks lpar_cmd upload_virt_logs generate_guest_asset_name get_guest_disk_name_from_guest_xml compress_single_qcow2_disk upload_supportconfig_log download_guest_assets is_installed_equal_upgrade_major_release);
+  = qw(update_guest_configurations_with_daily_build repl_addon_with_daily_build_module_in_files repl_module_in_sourcefile handle_sp_in_settings handle_sp_in_settings_with_fcs handle_sp_in_settings_with_sp0 clean_up_red_disks lpar_cmd upload_virt_logs generate_guest_asset_name get_guest_disk_name_from_guest_xml compress_single_qcow2_disk upload_supportconfig_log download_guest_assets is_installed_equal_upgrade_major_release generateXML_from_data);
```

Comment:
Please make it in  shorter lines. Now it is too long to be readable.

Reply:
I do not think this is quite necessary.